src/libraries/Common/src/Interop/Interop.ICU.iOS.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Globalization
+    {
+        [DllImport(Libraries.GlobalizationNative, EntryPoint = "GlobalizationNative_LoadICUData")]
+        internal static extern int LoadICUData(string? path);
```

Comment:
>? [](start = 53, length = 1)

Why this is nullable? I am seeing the implementation not expecting null values.

Reply:
Also, do we need a null terminated string or can we just use a `char*`?

Reply:
if it works, I have no preference.  What would you prefer?

Reply:
It cannot use `char*` - the unmanaged size expects UTF-8. `string` gives you UTF8 marshalling here.

I agree that it should not be nullable if the unmanaged side does not expect null.